### PR TITLE
Forecast concepts slides

### PR DESCRIPTION
**Pull request by sbfnk**

### CODE REVIEW

File: sessions/slides/forecasting-as-an-epidemiological-problem.qmd [hunk 1]
```diff
@@ -5,56 +5,81 @@ format:
...
+Maximise *sharpness* subject to *calibration*
 
-- how certain we are about certain future outcomes is important
-- providing uncertainty allows attributing probabilities to outcomes, i.e. X% chance of exceeding hospital capacity
-- Makes evaluation more complicated (but not impossible)
+- Statements about the future should be **correct** ("calibration")
+- statements about the future should aim to have **narrow uncertainty** ("sharpness")
 
-# Plan for the next few sessions
+### Plan for this two sessions
```
Comment (kathsherratt) [inline sessions/slides/forecasting-as-an-epidemiological-problem.qmd:80]:
```suggestion
### Plan for the next two sessions
```

Comment (sbfnk) [reply sessions/slides/forecasting-as-an-epidemiological-problem.qmd:80]: actually meant to say "plan for this session"

Comment (seabbs) [inline sessions/slides/forecasting-as-an-epidemiological-problem.qmd:80]:
```suggestion
### Plan for this session
```

**Review by seabbs — APPROVED** [review]
LGTM. I fixed the outstanding issue and resolved the comment

**Pull request merged**